apps/site/src/app/(home)/page.tsx

```diff
@@ -1,7 +1,15 @@
-import { Landing } from "./sections";
+import { Landing, MentorVolunteer } from "./sections";
 
 export default function Home() {
 	// Show landing section only if still in maintenance,
 	// otherwise show the rest of the sections
-	return process.env.MAINTENANCE_MODE_HOME ? <Landing /> : <Landing />;
+	return process.env.MAINTENANCE_MODE_HOME ? (
+		<Landing />
+	) : (
+		<>
+			<Landing />
+			<MentorVolunteer />
+			<div style={{ height: 400 }}></div>
```

Comment:
Question: What's the purpose of this `div`? Might be better to apply some padding or margin inside the `MentorVolunteer` component.

Reply:
Oh I was using this div to test the background earlier. It should have been removed.